subcommands/el2g/devices.go

```diff
@@ -37,14 +41,22 @@ fioctl el2g devices <device-id>
 #  04005001eee3ba1ee96e60047e57da0f6880
 # This ID is hexadecimal and must be prefixed in the CLI with 0x. For example:
 fioctl el2g devices --add 935389312472 0x04005001eee3ba1ee96e60047e57da0f6880
+
+# Delete a device with an se050
+fioctl el2g devices --del 935389312472 <device-id>
 `,
 	}
 	devicesCmd.Flags().StringVarP(&add, "add", "", "", "Whitelist the device for the given nc12 product id")
+	devicesCmd.Flags().StringVarP(&del, "del", "", "", "Unclaim device for the given nc12 product id")
```

Comment:
IMHO this creates the same confusion as the `--add`; besides, implementing these as mutually-exclusive flags makes the implementation more complex to understand or extend.